Source/WebInspectorUI/UserInterface/Views/GradientSlider.js

```diff
@@ -414,8 +404,10 @@ WI.GradientSliderKnob = class GradientSliderKnob extends WI.Object
             x = Math.min(Math.max(0, x), w);
         this.x = x;
 
-        if (this._detaching)
+        if (this._detaching) {
             this.y = event.pageY - this._startMouseY;
+            this._updateTransform();
+        }
         else if (this.delegate && typeof this.delegate.knobXDidChange === "function")
```

Comment:
Style: please put the `}` on the same line as the `else` (i.e. `} else if (...)`